source/modules/BookStore.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-trailing-spaces */
+
+import { bookContainer, bookTitle, bookAuthor } from './values.js';
+
+export default class AwesomeBook {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.bookList = [];
+  }
+
+  setBookList() {
+    this.bookList = JSON.parse(localStorage.getItem('book'));
+    return this.bookList;
+  }
+
+  // Remove Button
+  removeBook(btnAdderss, bookArray, btnIndex, parentDiv) {
+    btnAdderss.addEventListener('click', () => {
+      bookArray.splice(btnIndex, 1);
+      localStorage.setItem('book', JSON.stringify(bookArray));
+      parentDiv.remove();
+      window.location.reload();
+    });
+  }
+
+  // Adding Books
+  addBook() {
```

Comment:
- [ ] Kindly refactor your code to make the function to be arrow functions as required so kindly work on all the functions and refactor it to be arrow functions. 🙏🏻🔥💯

![image](https://user-images.githubusercontent.com/58553711/199234162-94ecb785-6acf-418a-b2d0-4219af57306c.png)